tests/test_rpm_metadata.py

```diff
@@ -97,7 +97,7 @@ def setUp(self):
                         os.path.join(self.tmpdir, os.path.basename(self.rpm)))
 
         proc = popen("createrepo", self.tmpdir)
-        self.assertIn("Workers Finished", proc.stdout)
+        self.assertIn(b"Workers Finished", proc.stdout)
```

Comment:
This test fails. With createrepo_c the output of the command createrepo differes. With python 3 it should test Pool finished and not Workers Finished. However we need to test Workers Finished for python2, so we need to check if one of those is present.

Reply:
@marusak Thank you, I have pushed the commit.